core/declarations.gms

```diff
@@ -182,6 +182,7 @@ p_cintraw(all_enty)                                  "carbon intensity of fossil
 p_CapFixFromRWfix(ttot,all_regi,all_te)              "parameter for fixing capacity variable to Real-World values in 2010/2015"
 p_deltaCapFromRWfix(ttot,all_regi,all_te)            "parameter with resulting deltacap values resulting from fixing capacity to real-world values in 2010/2015"
 
+p_adjCostInv(ttot,all_regi,all_te)                   "adjustment cost to technology investment"
```

Comment:
I think the description can be confused with the actual parameters for adjustment cost. Better say something like "marginal adjustment cost calculated in postsolve for diagnostics".

Reply:
haha good point, it is changed to your suggestion. I don't think it's too jargon-y :P